DeployToAzureAppService.md

```diff
@@ -0,0 +1,94 @@
+# Deploy this web app to Azure App Service
+
+You can deploy this web application to Azure App Service with the instructions below. If you don't have an Azure account, you can [open a free account](https://azure.microsoft.com/en-us/free/). Once everything is configured, the changes you commit to your repo will be automatically deployed to Azure App Service.
+
+## Requisites
+
+* Create an [Azure account](https://azure.microsoft.com/)
+* [Install the Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli)
+* Login in [Azure Portal](https://portal.azure.com)
+* [Enable Cloud Shell](https://learn.microsoft.com/azure/cloud-shell/get-started) to your Azure subscription
+
+## Deployment steps
+
+Run the commands below in a new Powershell terminal. Make sure that the Azure CLI is installed by running `az --version`
+
+### MySQL database creation
+
+This simple web app uses MySQL to store the book and users data. The steps in this section will help you create a MySQL server in Azure.
+
+```powershell
+
+# Configuration
+
+$SUFFIX="$(Get-Random -Minimum 100000 -Maximum 999999)"
+$RESOURCE_GROUP="onlinebookstore${SUFFIX}-rg"
+$LOCATION="eastus2"
+$MYSQL_ADMIN_USER="azureuser"
+$MYSQL_ADMIN_PASSWORD="adminPass"
+$WEBAPP_NAME="onlinebookstore${SUFFIX}"
+$MYSQL_HOST="onlinebookstore${SUFFIX}mysql"
+
+# Create resource group for all resources
+
+az group create --name $RESOURCE_GROUP --location $LOCATION
+
+# Create a MySQL flexible server
+
+az mysql flexible-server create --resource-group $RESOURCE_GROUP --name $MYSQL_HOST --location $LOCATION --admin-user $MYSQL_ADMIN_USER --admin-password $MYSQL_ADMIN_PASSWORD --public-access 0.0.0.0 --sku-name Standard_B1ms
+```
+
+Once the previous step completes, we'll create the database by logging into Azure Portal and connect to the new MySQL server.
+
+Run the following command in the same terminal as earlier to find the command to use to connect to the database:
+
+```powershell
+echo "Connect with: mysql -h ${MYSQL_HOST}.mysql.database.azure.com -u azureuser -p"
+```
+
+Now, while logged into Azure Portal, start a Cloud Shell and run the `mysql` command from the previous step. When prompted, enter the password defined in the `Configuration` section.
+
+In a text editor open the file `setup/ExampleData.sql`. Copy the contents of the file and paste it in the Cloud Shell terminal where you are connected to MySQL. The SQL commands should complete successfully. Enter `exit` to exit the MySQL prompt. You can close the Cloud Shell terminal when done.
+
+### Web app creation
+
+Run the following commands to create a new web application running Tomcat 9 with Java 11:
+
+```powershell
+az appservice plan create --resource-group $RESOURCE_GROUP --name "${WEBAPP_NAME}-asp" --sku P1V3 --is-linux --location $LOCATION

```

Comment:
Do we really need a premium SKU at least to run this? 
Just wondering since this is demo code, whether a lower SKU like Standard would also be sufficient.

Reply:
Good point, I'll test it with a lower SKU, like a Basic B1 or B2.

Reply:
Updated in the latest commit.